index.js

```diff
@@ -559,7 +595,6 @@ const createJavaBom = async (
         }
       }
     }
-    pkgList = await utils.getMvnMetadata(pkgList);
```

Comment:
Can you add this back please since other users of this tool would suddenly loose some information?

Reply:
yes, brought it back